bin/build-templates.js

```diff
@@ -0,0 +1,35 @@
+const {promisify} = require('util');
+const fs = require('fs');
+const path = require('path');
+require('dotenv').config();
+
+const inputPath = path.resolve(__dirname, './../templates');
+const outputPath = path.resolve(__dirname, './../dist/templates');
+const readdir = promisify(fs.readdir);
+const readFile = promisify(fs.readFile);
+const writeFile = promisify(fs.writeFile);
+const mkdir = promisify(fs.mkdir);
+const replacements = {
+	'{{DOCS_RELEASE_BUCKET}}': process.env.DOCS_RELEASE_BUCKET,
+	'{{LAMBDAS_RELEASE_BUCKET}}': process.env.LAMBDAS_RELEASE_BUCKET
+};
+
+/**
+ * Read in each .template from the templates directory.
+ * Then rewrite all replacements with the appropriate value.
+ * Save rewritten .template files to dist/templates directory.
+ */
+mkdir(outputPath)
```

Comment:
Where would a failure be caught?

Reply:
Good call, I'll push an update to address this.